src/dbi.cpp

```diff
@@ -56,107 +56,6 @@ DbiWrap::~DbiWrap() {
     }
 }
 
-
-class PutWorker : public Nan::AsyncWorker {
```

Comment:
Since you just added this in the previous commit, and you're deleting it right away, it would make sense to squash these two commits.